File: c++/codes/L2_public.cpp
#include <iostream>
#include <string>

class Teacher {
public:
    std::string name;
    std::string department;

    void print() {
        std::cout << "Name: " << name << ", Department: " << department << std::endl;
    }
};

int main() {
    Teacher teacher1;
    teacher1.name = "Dr. Smith";
    teacher1.department = "Computer Science";
    teacher1.print();
    return 0;
}

File: python/codes/L2_public.py
class Teacher:
    def __init__(self, name, department):
        self.name = name
        self.department = department

    def print_details(self):
        print(f"Name: {self.name}, Department: {self.department}")

teacher1 = Teacher("Dr. Smith", "Computer Science")
teacher1.print_details()
